apps/web/src/components/payment-methods-form/index.tsx

```diff
@@ -8,13 +8,22 @@ import AddMethodsSuccess from './sections/add-methods-success'
 
 import Loading from '@/components/loading/loading'
 import {
+  validateBankAccount,
+  validateBidsForm,
+  validateBidsFormForWires,
   validateExpirationDate,
   validatePurchaseForm,
 } from '@/utils/purchase-validation'
 
 export interface MyProfilePaymentMethodsAddProps {
   formErrors?: ExtractError<
-    ReturnType<typeof validatePurchaseForm | typeof validateExpirationDate>
+    ReturnType<
+      | typeof validateBankAccount
+      | typeof validateBidsForm
+      | typeof validateBidsFormForWires
+      | typeof validatePurchaseForm
+      | typeof validateExpirationDate
+    >
```

Comment:
The changes in unrelated files are because we're sharing the `formErrors` validation.